Headers/AppKit/NSScrubberLayout.h

```diff
@@ -0,0 +1,93 @@
+/* Definition of class NSScrubberLayout
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: Wed Apr  8 09:20:18 EDT 2020
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Library General Public License for more details.
```

Comment:
Looks like your template is again wrong?

Reply:
Yep, changes to the generation script.

Reply:
Fixed here and in the template scripts.  There are separate scripts for GUI and BASE and also for classes and templates.  I have corrected them all so this shouldn't come up again.

Reply:
no